web-app-demo/Backend/api/helper_test.py

```diff
@@ -71,14 +71,12 @@ def test_process_document_normal(self, process_document_mock):
             "location": LOCATION,
             "file_path": os.path.join(__location__, "test_docs/file"),
             "file_type": "application/pdf",
-            "processor_id": PROCESSOR_ID
+            "processor_id": PROCESSOR_ID,
```

Comment:
Not Sure why these files got included.

Can the web-app-demo files be taken out of this PR?

Reply:
Looks like owlbot was trigger to run when the PR was opened and added these changes. I'll try and figure out how to turn off owlbot for this PR